docs/arch_develop.md

```diff
@@ -277,6 +277,63 @@ Commands and events are defined on the backend via sub-classes of `VueWebview`.
     client.init(data => (this.data = data ?? this.data))
     ```
 
+## Webviews (non Vue)
+
+Some webviews (amazon q chat view, codewhisperer security panel) rely on the native vscode webview implementation using the `vscode.WebviewViewProvider` and the `vscode.window.registerWebviewViewProvider(viewType, panel)` extension APIs. They follow the standard structure given by the webview documentation: https://code.visualstudio.com/api/extension-guides/webview.
+
+### Importing css
+
+css imports for non vue webviews should be imported when generating the webview provider html, rather than loaded inside of the javascript. This allows the javascript to be used in e2e tests:
```

Comment:
should we mention here the specific error that appeared and was addressed by https://github.com/aws/aws-toolkit-vscode/pull/4804 ? that will make this easy to find if that symptom occurs again.